ldaptor/protocols/pureldap.py

```diff
@@ -612,7 +612,7 @@ class LDAPFilter_present(LDAPAttributeDescription):
     tag = CLASS_CONTEXT | 0x07
 
     def asText(self):
-        return '(%s=*)' % self.value
+        return '(%s=*)' % to_unicode(self.value)
```

Comment:
Can you test this in py2 with a string with unicode chars? I think it will get an error.

Reply:
Do we still want to support PY2 ?

Reply:
this is not our library. this is ldaptor, and I think they still support py2.

Reply:
Tested with PY2 and discussed with Aren over slack